src/components/events/createEvent/EventsGeneralInfo.jsx

```diff
@@ -16,7 +16,7 @@ import {
 } from 'antd';
 import useViewPort from '../../../common/useViewPort';
 import EventTypeModal from './EventTypeModal';
-import './createEvent.css';
+import styles from './CreateEvent.module.css';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module './CreateEvent.module.css'.